properties/P000133.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: P000133
+slug: linearly_orderable
+name: Linearly orderable
+aliases:
+  - Linearly ordered topological space
+  - LOTS
+refs:
+  - wikipedia: Order_topology
+    name: Order topology
+  - mr: '3728284'
+    name: Topology (Munkres)
+  - mr: MR2048350
+    name: General Topology (Willard)
+  - mr: MR1039321
+    name: General Topology (Engelking, 1989)
+---
+
+A space whose topology can be induced by a total order on the underlying set.
+
+Defined on page 84 of {{mr:3728284}}, in 6D of {{mr:MR2048350}}, and 1.7.4 of {{mr:MR1039321}}.
```

Comment:
I think we should also mention S&S, something like "Also defined in counterexample #39 ("Order Topology") in {{doi:10.1007/978-1-4612-6290-9}}."